src/services/github.js

```diff
@@ -31,14 +31,15 @@ const fetchDevices = async () => {
 const fetchBuilds = async codename => {
   try {
     const res = await request(`${baseURL}/OTA/11/${codename}.json`);
-
+    console.log(res);
```

Comment:
should i delete it XD?

Reply:
Yus, why put stuff in console anyways. it will create spam for us while debugging.